lib/system_description.rb

```diff
@@ -276,8 +276,20 @@ def runs_service?(name)
   end
 
   def has_file?(name)
-    self["config_files"].files.any? { |file| file.name == name } ||
+    found = self["config_files"].files.any? { |file| file.name == name } ||
       self["unmanaged_files"].files.any? { |file| file.name == name }
+    return true if found
+
+    file = self["unmanaged_files"].files.find { |file| file.name == File.join(File.dirname(name), '') }
```

Comment:
Shadowing outer local variable - `file`.<br>Line is too long. [103/100]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.